examples/resources/flagsmith_feature_state/import.sh

```diff
@@ -1 +1 @@
-terraform import flagsmith_feature_state.some_flag <enviroment_client_key>,<feature_name>
+terraform import flagsmith_feature_state.some_flag <enviroment_client_key>,<feature_state_uuid>
```

Comment:
Yeah, we need it because it's part of the schema, and we need to save it in state, and there is a no way to get environment_client key from feature_state_uuid, unless we update/create a new endpoint